addons/stock/report/product_templates.xml

```diff
@@ -2,42 +2,119 @@
 <odoo>
     <data>
         <template id="label_product_product_view">
-		<t t-foreach="quantity.items()" t-as="barcode_and_qty_by_product">
+            <t t-foreach="quantity.items()" t-as="barcode_and_qty_by_product">
                 <t t-set="product" t-value="barcode_and_qty_by_product[0]"/>
                 <t t-foreach="barcode_and_qty_by_product[1]" t-as="product_info">
                     <t t-set="barcode" t-value="product_info['barcode']"/>
                     <t t-set="currency_id" t-value="pricelist.currency_id or product.currency_id"/>
                     <t t-foreach="range(product_info['quantity'])" t-as="qty">
                         <t t-translation="off">
-^XA^CI28
-^FT100,80^A0N,40,30^FD<t t-out="product_info['display_name_markup']"/>^FS
-<t t-if="product.default_code and len(product.default_code) &gt; 15">
-^FT100,115^A0N,30,24^FD<t t-out="product_info['default_code'][0]"/>^FS
-^FT100,150^A0N,30,24^FD<t t-out="product_info['default_code'][1]"/>^FS
-</t>
-<t t-else="">
-^FT100,150^A0N,30,24^FD<t t-out="product_info['default_code'][0]"/>^FS
-</t>
-<t t-if="price_included">
-^FO600,100,1
-<t t-if="currency_id.position == 'after'">
-^A0N,66,48^FH^FD<t t-esc="pricelist._get_product_price(product, 1, currency_id)" t-options='{"widget": "float", "precision": 2}'/><t t-esc="currency_id.symbol"/>^FS
-</t>
-<t t-if="currency_id.position == 'before'">
-^A0N,66,48^FH^FD<t t-esc="currency_id.symbol"/><t t-esc="pricelist._get_product_price(product, 1, currency_id)" t-options='{"widget": "float", "precision": 2}'/>^FS
-</t>
-</t>
-<t t-if="barcode">
-^FO100,160^BY3
-^BCN,100,Y,N,N
-^FD<t t-out="barcode"/>^FS
-</t>
-^XZ
+                            ^XA^CI28
+
```

Comment:
Be warned all added spaces will be in the .txt output.
I would keep like it was before even if it's ugly in the IDE.
Same comment about additional line break
```suggestion
^XA^CI28
```
It's not so much (~ a bit more than 100 bytes) but with no extra linebreak and no all those spaces, the file size is half as heavy. Since we send that through the API, I think it's good to take

Reply:
I agree with you, but to me the benefits of having the clear indentation outweigh the negatives.

As such, I leave the choice to you :)

I will remove the indentation if you think it's better